src/components/panel/panel.spec.js

```diff
@@ -597,12 +598,13 @@ describe('$mdPanel', function() {
 
     it('should allow locals to be injected in the controller', function() {
```

Comment:
optional: Add a test that a panel can close itself. That's the main benefit (I think) of injecting MdPanelRef.